draftlogs/7115_fix.md

```diff
@@ -0,0 +1,2 @@
+ - Update turf.js to v7 [[#7115](https://github.com/plotly/plotly.js/pull/7115)]
+ 
```

Comment:
Please add a thank you message to your github profile similar to those found in the changelog.
Merci beaucoup!